apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,38 @@
+#include "TutorialServo.h"
+#include <cmath>
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs)
+              : m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs)
+{
+    // Initialize servo pin and period
+    m_servoPwmOut(servoPin);
+    m_servoPwmOut.period_ms(m_periodInMs);
+
+}
+
+// Set servo position (ex. 45deg)
+void TutorialServo::setPositionInDegrees(const float degrees){
```

Comment:
uhh this works too, but id rather you just clamp the input to put it inside the range. if the range is 0 to 180 and someopne passes in 200 just put  it to 180

Reply:
Oh lol that makes sense, I thought you were looking for a more big brain solution